source/support.tex

```diff
@@ -635,6 +635,7 @@
   // also in \libheader{unordered_map}, \libheader{unordered_set}
 #define @\defnlibxname{cpp_lib_hardware_interference_size}@        201703L // also in \libheader{new}
 #define @\defnlibxname{cpp_lib_has_unique_object_representations}@ 201606L // also in \libheader{type_traits}
+#define @\defnlibxname{hazard_pointer}@                            202306L // also in \libheader{hazard_pointer}
```

Comment:
I don't see this proposed in the paper.
```suggestion
#define @\defnlibxname{cpp_lib_hazard_pointer}@                    202306L // also in \libheader{hazard_pointer}
```
```suggestion
```

Reply:
Maybe it's because you can just detect the presence of the header `<hazard_pointer>`.

Reply:
Although that's not necessarily reliable,
as implementations provide empty headers when compiling with earlier standards.
I read something about that recently, but I can't find the source.

Reply:
It turns out, like the other RCU paper,
that the proposed feature-test macro is very far from the wording.
§1.1 vs. §4.